2 Целовальников

darthogster opened the pull request.

Commit:
Assignment 2

darthogster committed:
Merge branch 'intro' into 2_lastname

odinkomnogim commented:
Поле 'is_green' отсутствует в итоговом файле, хотя в коде оно добавляется.
Для дебагинга можно воспользоваться:
```
layer_defn = layer.GetLayerDefn()
for field in range(layer_defn.GetFieldCount()):
    field_definition = layer_defn.GetFieldDefn(field)
    print (field_definition.name)
```
чтобы понять, где оно устанавливается, а где, почему и в какой момент пропадает перед сохранением

darthogster committed:
README.md

Посмотрел код,и результат,вроде в файле добавляется "is_green no", в QGIS изменения видны

darthogster commented:
Здраствуйте,у меня в файле is_green при этом же коде есть и отображается,что в самом файле,что и в QGIS

darthogster commented:
![image](https://github.com/user-attachments/assets/d36ffe61-7d5f-4177-a261-6b1d0136afaf)
![image](https://github.com/user-attachments/assets/480520d9-cf80-4408-8d22-c90a89c10555)

odinkomnogim commented:
Upd:
Перезапустила - действительно есть нужное поле, прошу прощения, не туда посмотрела
2
